src/ParkMap.js

```diff
@@ -23,15 +24,14 @@ class ParkMap extends Component {
       selectedPark: null
     });
   }
-
-  showParksByState = (state) => {
-    //this.setstate = currentParksToShow should now only include parks from parameter of state
-  }
   
   render() {
+    let stateZoom = this.state.uniqueStateZoom[this.props.stateName] ? this.state.uniqueStateZoom[this.props.stateName] : 6;
     return (
      <div className="map-container">
-        <Map id="map" center={this.state.position} zoom={this.state.zoom}>
+        <Map id="map" center=
+          { this.props.stateCoord.length > 0 ? this.props.stateCoord : this.state.position }
```

Comment:
Great use of ternary operator here and above for the coordinates and zoom level.